lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -606,8 +607,7 @@ def getvm(self, name=None, uuid=None, folder=None, name_match=None, cache=False)
                 paths = [x for x in paths if x.endswith(self.params['folder'])]
                 if len(paths) > 1:
                     self.module.fail_json(
-                        msg='%s matches more than one folder. Please use the absolute path starting with /vm/' %
-                            self.params['folder'])
+                        msg='%(folder)s matches more than one folder. Please use the absolute path starting with /vm/' % self.params)
```

Comment:
Well, de PEP8 line-length was disabled from the checks. 80 columns by today's standards is no longer relevant and IIRC discussions wrt. this were not as strict. Besides the file is full of lines over the 80 column limit, I don't think this is a real concern. is it ?

    [dag@moria modules]$ awk '{ print length }' cloud/vmware/vmware_guest.py | sort | uniq -c | sort -n -k2 | tail
      4 114
      4 115
      2 116
      3 117
      1 118
      1 119
      1 120
      2 133
      1 142
      1 172